### PR TITLE
add omnitone as a dependency to three CNE-2372

### PR DESCRIPTION
**Pull request by ekelak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,12 @@
     "argparse": "^1.0.3",
     "jscs": "^1.13.1",
     "rollup": "^0.36.3",
+    "rollup-plugin-commonjs": "^5.0.5",
+    "rollup-plugin-node-resolve": "^2.0.0",
     "rollup-watch": "^2.5.0",
     "uglify-js": "^2.6.0"
+  },
+  "dependencies": {
+    "omnitone": "^0.1.6"
```
Comment (mjprude) [inline package.json]: This makes me nervous. This is a wider conversation we should have, but I feel like being more permissive than `~` on our node modules is asking for trouble.

File: rollup.config.cne.js [hunk 2]
```diff
@@ -29,7 +32,23 @@ export default {
...
+		glsl(),
+		nodeResolve({ jsnext: true, main: true }),
+		commonjs({
+			// non-CommonJS modules will be ignored, but you can also
+			// specifically include/exclude files
+			include: 'node_modules/**',  // Default: undefined
+
+			// search for files other than .js files (must already
+			// be transpiled by a previous plugin!)
+			extensions: [ '.js', '.coffee' ],  // Default: [ '.js' ]
```
Comment (ScottLNorvell) [inline rollup.config.cne.js]: i have a feeling you don't need `'.coffee'`

Comment (mjprude) [inline rollup.config.cne.js]: Do we need `.coffee` here? Also some weird indenting in this block.

Comment (ekelak) [reply rollup.config.cne.js]: yup no need for coffee. I keep trying to fix the indent, but it fails! wth!

Comment (ekelak) [reply rollup.config.cne.js]: 👍

File: package.json (hunk 1 above)
Comment (ekelak) [reply package.json]: 👍

Comment (ScottLNorvell) [reply package.json]: yeah let's lock in a version and then we can be sure to add this to our updates list

**Pull request merged**